content/webapp/pages/stories.tsx

```diff
@@ -99,12 +101,18 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     // would it be faster to skip all the fetchLinks?  Is that possible?
     const storiesPagePromise = fetchPage(client, prismicPageIds.stories);
 
+    const featuredSerial =
+      await client.client.getSingle<FeaturedSerialPrismicDocument>(
+        'featured-serial'
+      );
+    const featuredSerialId =
+      (isFilledLinkToDocument(featuredSerial?.data?.serial) &&
+        featuredSerial.data.serial.id) ||
```

Comment:
Let's push this down into a `transformFeaturedSerial` method, to keep the Prismic document-handling logic out of this file.